README.md

```diff
@@ -340,6 +340,9 @@ public interface ExampleHandler {
 }
 ```
 
+For this example, the generated class will respect `ExecutableHandler` interface
```

Comment:
Because we have a look at extending ExecutableHandler in the example above the description. I can remove it, if you think it is redundant.